src/logger/fileWriter.c

```diff
@@ -229,14 +231,28 @@ static void flushLogfile(FIL *file)
     }
 }
 
-static int openNewLogfile(char *filename)
+static void file_led_set(bool on) {
```

Comment:
LED 3 is the error LED. function should be called enable_file_error_indicator or something similar and succinct.